legacy/src/_shared/js/dom.js

```diff
@@ -23,7 +23,6 @@ let img = createElement('img');
 let div = createElement('div');
 
 export {
-    closest,
```

Comment:
Dom was adding this for testing but when we paired he told me to remove it because it was causing an error.